addons/website/views/website_templates.xml

```diff
@@ -1553,7 +1553,7 @@
                             <h5>Follow us</h5>
                             <ul class="list-unstyled">
                                 <li class="py-1"><i class="fa fa-1x fa-fw fa-facebook-square mr-2"/><a href="/website/social/facebook" target="_blank">Facebook</a></li>
-                                <li class="py-1"><i class="fa fa-1x fa-fw fa-twitter-square mr-2"/><a href="/website/social/twitter" target="_blank">Twitter</a></li>
+                                <li class="py-1"><i class="fa fa-1x fa-fw fa-twitter mr-2"/><a href="/website/social/twitter" target="_blank">Twitter</a></li>
```

Comment:
I'll remove this diff, the value is really low in stable, especially 15 (just a tiny bit more beautiful).
But the risk is not null, seems likely that someone xpath'd this element to add a custom social network after it or to hide it or whatever.

Not worth the risk in stable